packages/tracing/src/node/integrations/apollo.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @sentry-internal/sdk/no-optional-chaining */
```

Comment:
Can we add this rule to eslintrc.js for all files under `integrations/node`? IMHO cleaner than to have this in the file(s) here.

Reply:
yes will change!